pytorch_lightning/core/saving.py

```diff
@@ -327,22 +327,27 @@ def save_hparams_to_yaml(config_yaml, hparams: Union[dict, Namespace]) -> None:
     if not os.path.isdir(os.path.dirname(config_yaml)):
         raise RuntimeError(f'Missing folder: {os.path.dirname(config_yaml)}.')
 
-    if Container is not None and isinstance(hparams, Container):
-        from omegaconf import OmegaConf
-        OmegaConf.save(hparams, config_yaml, resolve=True)
-        return
-
-    # saving the standard way
+    # convert Namespace or AD to dict
     if isinstance(hparams, Namespace):
         hparams = vars(hparams)
     elif isinstance(hparams, AttributeDict):
         hparams = dict(hparams)
-    assert isinstance(hparams, dict)
 
+    # saving with OmegaConf objects
+    if OmegaConf is not None:
```

Comment:
this will crash because not everyone has OmegaConf installed.

Reply:
My above fix should also fix this! Just ran without OmegaConf installed and it passed the tests.